pysat/ssnl/avg.py

```diff
@@ -65,7 +65,9 @@ def median1D(const, bin1, label1, data_label, auto_bin=True, returnData=False):
     if isinstance(const, pysat.Instrument):
         const = [const]
     elif not isinstance(const, pysat.Constellation):
```

Comment:
Recommend adding a second evaluation for `pysat._instrument.Instrument`, etc.

Reply:
Will do. As far as I know that string is correct. If I check things locally, 
```
In [7]: test = pysat.Instrument('pysat', 'testing',
   ...:                         clean_level='clean',
   ...:                         orbit_info={'index': 'mlt'},
   ...:                         update_files=True)
/Users/russellstoneback/opt/miniconda3/envs/standard2/bin/ipython:4: DeprecationWarning: Instrument attributes [units_label, name_label, notes_label, desc_label, min_label, max_label, fill_label] will be accessible through `Instrument.meta.labels` in pysat 3.0.0
  import re
/Users/russellstoneback/opt/miniconda3/envs/standard2/bin/ipython:4: DeprecationWarning: Meta labels [plot_label, axis_label, scale_label] are no longer standard metadata quantities in pysat 3.0.0
  import re
/Users/russellstoneback/opt/miniconda3/envs/standard2/bin/ipython:4: DeprecationWarning: Strict times will eventually be enforced upon all instruments. (strict_time_flag)
  import re

In [8]: type(test)
Out[8]: pysat._instrument.Instrument

In [9]: isinstance(test, pysat.Instrument)
Out[9]: True
```

It works out. The line in ssnl that is causing the problem is:

```
    if isinstance(const, pysat.Instrument):
        const = [const]
    elif not isinstance(const, pysat.Constellation):
        estr = ''.join(("Parameter must be an Instrument or a Constellation.",
                        " Received: ", str(type(const))))
        raise ValueError(estr)
```
the `isinstance(const, pysat.Instrument)` should evaluate to True like it does in In[9] above. I think this may actually be a weird side effect of reloading pysat in the new class wrapped module regime. It is possible the `pysat` that was originally attached to the ssnl module when it was first imported is now a different instance of `pysat` in the namespace that created the test instrument supplied to the method..... I can probably test that.